src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -137,6 +145,27 @@ protected JestElasticsearchClient(Map<String, String> props, JestClientFactory f
             .preemptiveAuthTargetHosts(address.stream()
                 .map(addr -> HttpHost.create(addr)).collect(Collectors.toSet()));
       }
+
+      if (secured) {
+        log.info("Using secured connection");
```

Comment:
Do we want to include the URLs in this log message? Seems like it'd be more useful, especially when using multiple ES connectors in the same Connect cluster.

```suggestion
        log.info("Using secured connection to {}", config.urls());
```